### DISCUSSION TITLE
Will the promptflow vscode extension be open sourced?

### DISCUSSION THREAD

**Discussion by ybz21** [microsoft/promptflow#1347] [General]
Will the promptflow vscode extension be open sourced?

**Comment by ybz21** [comment]
We'd like to make more function for this extension

**Comment by JYC-99** [comment]
There is no plan to open source the ux parts so far. Both web experience and vscode extension code are still in private repos.
We have the library to draw the DAG graph on the GUI public: https://github.com/microsoft/react-dag-editor
For other questions, bug reports or feature suggestions, please open issues and the crew will track them.
